### DISCUSSION TITLE
GitHub does not support Microsoft X++ language

### DISCUSSION THREAD

**Discussion by gbamqzkdyg** [community/community#62644] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug

### Body

I tried to search the code written in X++ language and realized that it is not available in the filter:

![image](https://github.com/community/community/assets/29323147/4361d8e4-eaba-4ebf-a399-62737795528f)

I understand that the list of the language will not include all languages but only the popular ones. Based on the TIOBE Index for July 2023, the X++ language is ranked as #49, which looks "popular".

Will GitHub consider including the X++ language also in the language list?

![image](https://github.com/community/community/assets/29323147/ace62b75-45bc-4b90-8f01-1c7864a80944)

**Comment by look** [comment, marked as answer]
You can request a new language be added by [filing an issue in the Linguist repository](https://github.com/github-linguist/linguist/issues/new?assignees=&labels=Add+Language%2C+Good+First+Issue&projects=&template=new_language.md&title=).
